tasktiger/redis_scripts.py

```diff
@@ -450,7 +450,7 @@ def get_expired_tasks(self, key_prefix, time, batch_size, client=None):
         # [queue1, task1, queue2, task2] -> [(queue1, task1), (queue2, task2)]
         return list(zip(result[::2], result[1::2]))
 
-    def execute_pipeline(self, pipeline, client=None):
+    def execute_pipeline(self, pipeline):
```

Comment:
Let's keep the `client` arg to keep it consistent with the other scripts, and with pyredis' `Script.__call__` function, and to possibly allow third party code to run this with a different Redis client instance.

We could just say `(client or self.redis).pipeline()` below (or `if not client: client = self.redis`).